### PR TITLE
openflow_acl_test.accessed_deny_all_target.telnet: remove if_port

### PR DESCRIPTION
**Pull request by quanwenli**
1. When add if_port option in ovs, the drop function does not work
2. the default bridge name is atbr0

Signed-off-by: Wenli Quan <wquan@redhat.com>

ID:1467827

### CODE REVIEW

**Comment by vivianQizhu** [comment]
@YanhuiMa @xiywang Please help review, thanks.

File: qemu/tests/openflow_acl_test.py [hunk 1]
```diff
@@ -174,9 +174,8 @@ def access_service(access_sys, access_targets, disabled, host_ip,
                     test.fail(err_msg)
                 session.close()
 
-    def get_acl_cmd(protocol, in_port, action, extra_options):
+    def get_acl_cmd(protocol, action, extra_options):
```
Comment (xiywang) [inline qemu/tests/openflow_acl_test.py]: in_port is valid for drop actions.
but the value of in_port should be modified.
try this cmd in host: ovs-ofctl add-flow ovs0 tcp,in_port=$guest1_tap(eg: tap0),nw_dst=$guest2_ip,tp_src=23,actions=drop

**Comment by xiywang** [comment]
The code should works fine with #1427 and brname changed to atbr0.
No need to del in_port.

(1/1) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.x86_64.myprovider.openflow_acl_test.accessed_deny_all_target.telnet: PASS (500.38 s)

**Comment by vivianQizhu** [comment]
If I understand correctly this patch could be closed right?

**Comment by quanwenli** [comment]
@vivianQizhu ,no , 1427 should be closed since I have update private bridge to 1427. 
For 1480, the I will update in_port part.

**Comment by xiywang** [comment]
You can try the upstream code, with the patch #1427 (no need to apply this patch of #1480) and run this case again. It should pass.

**Comment by quanwenli** [comment]
Ｉ tried with upstream with patch 1427. still failed, we need to remove if_port parameter. 

AIL 1-Host_RHEL.m7.u6.qcow2.virtio_blk.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.openflow_acl_test.accessed_deny_all_target.telnet -> TestFail: Still can access localhost after disable it from ovs. Command: telnet. Output:  Password: 
Last login: Wed Sep 26 04:38:19 from 192.168.58.105

**Comment by quanwenli** [comment]
I am using openvswitch-2.9.0-55.el7fdp.x86_64.rpm

**Comment by quanwenli** [comment]
Depends on avocado-framework/avocado-vt#1579

**Comment by xiywang** [comment]
Tested again, do not need to drop in_port.

On host:
# ovs-vsctl show
.......
    Bridge privatebr
        Port privatebr
            Interface privatebr
                type: internal
        Port "tap1"
            Interface "tap1"
        Port "tap0"
            Interface "tap0"
    ovs_version: "2.10.0"

# ovs-ofctl add-flow privatebr tcp,in_port=tap0,tp_src=23,actions=drop

guest1 cannot access host after acl protocol drop port 23 packets from tap0.

The env is on 10.73.224.26 if you needed.

**Comment by quanwenli** [comment]
I am closing this pr since do not need anymore, thanks all

**Pull request closed without merging**